doc/cli.md

```diff
@@ -0,0 +1,230 @@
+   :presence \"included\"}
+ failed 'any' property: evaluated values must include some values given by 'any'
+ statement values:
+   The first Activity
+
+----- Statement Validation Failure -----
+Template ID:  https://xapinet.org/xapi/yet/calibration/v1/templates#activity-3
+Statement ID: 00000000-4000-8000-0000-000000000000
+
+Template rule was not followed:
+  {:any [\"https://xapinet.org/xapi/yet/calibration/v1/concepts#activity-3\"],
+   :location \"$.object.id\",
+   :presence \"included\"}
+ failed 'any' property: evaluated values must include some values given by 'any'
+ statement values:
+   https://xapinet.org/xapi/yet/calibration/v1/concepts#activity-1
+
+Template rule was not followed:
+  {:any [\"Activity 3\"],
+   :location \"$.object.definition.name.en-US\",
+   :presence \"included\"}
+ failed 'any' property: evaluated values must include some values given by 'any'
+ statement values:
+   Activity 1
+
+Template rule was not followed:
+  {:any [\"The third Activity\"],
+   :location \"$.object.definition.description.en-US\",
+   :presence \"included\"}
+ failed 'any' property: evaluated values must include some values given by 'any'
+ statement values:
+   The first Activity
+
+-----------------------------
+Total errors found: 7
+```
+
+If we set the `--short-circuit` flag along with `--all-valid`:
+```
+% ./bin/persephone.sh validate \
+  --profile sample_profiles/calibration.jsonld \
+  --statement sample_statements/calibration_1.json \
+  --short-circuit --all-valid
+```
+then only the first validation failure encountered will appear.
+
+We can use the `--template-id` argument to select which Templates the Statement is validated against.
+```
+% ./bin/persephone.sh validate \
+  --profile sample_profiles/calibration.jsonld \
+  --statement sample_statements/calibration_1.json \
+  --template-id https://xapinet.org/xapi/yet/calibration/v1/templates#activity-2
+  --template-id https://xapinet.org/xapi/yet/calibration/v1/templates#activity-3
+```
+This way, we pass to the CLI the two Templates the Statement is invalid against, so running this will result in the same validation failure output as in the previous example.
+
+Note that if we set `--template-id` to the ID of a non-existent Template, e.g.
+```
+% ./bin/persephone.sh validate \
+  --profile sample_profiles/calibration.jsonld \
+  --statement sample_statements/calibration_1.json \
+  --template-id http://random-template.org
+```
+the validation will pass vacuously, as there are technically no Templates the Statement is invalid against.
+
+## Examples for `persephone match`
+
+We can match the Statement batch consisting of `calibration_1.json` and `calibration_2.json` against the Patterns in the `calibration.jsonld` Profile as so:
+```
+% ./bin/persephone.sh match \
+  --profile sample_profiles/calibration.jsonld \
+  --statement sample_statements/calibration_1.json \
+  --statement sample_statements/calibration_2.json
+```
```

Comment:
I note that I can just send `calibration_1.json` and get a match. Is this because the pattern in the calibration profile will successfully terminate with just one, or because it will pass with a partial match (a la xapipe)?

Reply:
It's a partial match

Reply:
Gotcha. Given that it would probably be good to give the user path output regardless of success so they can disambiguate, see https://github.com/yetanalytics/project-persephone/pull/39/files#r1178344643